test_e2e/env_setup/duckdb/create-duckdb-instance.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+DATABASE_NAME=$1
+DATABASE_FILE_PATH=$2
+
+# Create the DuckDB instance
+touch "$DATABASE_FILE_PATH"
```

Comment:
This line caused problems for me. Worked better when commented out.

Reply:
makes sense.